recipes-browser/wpebackend-fdo/wpebackend-fdo/PKG_CONFIG_SYSROOT_DIR_for_wayland_scanner.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8a94387..5cb9a38 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -94,9 +94,9 @@ add_custom_command(
+            ${CMAKE_BINARY_DIR}/bridge/wpe-bridge-client-protocol.h
+            ${CMAKE_BINARY_DIR}/bridge/wpe-bridge-server-protocol.h
+     DEPENDS ${CMAKE_SOURCE_DIR}/src/bridge/wpe-bridge.xml
+-    COMMAND ${WAYLAND_SCANNER} ${WAYLAND_SCANNER_CODE_ARG} < ${CMAKE_SOURCE_DIR}/src/bridge/wpe-bridge.xml > ${CMAKE_BINARY_DIR}/bridge/wpe-bridge-protocol.c
```

Comment:
This looks more like a workaround :( Why is WAYLAND_SCANNER not properly set in the first place?

Reply:
Can you check if wayland-native is correctly added to DEPENDS ?

Reply:
Yes, this is only happening in a particular environment. 


I'm proned to think this MR is unnecessary

Reply:
Me too :) Seems more like an issue in your specific environment.